guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -406,6 +406,16 @@ You can now create an image from this virtual machine.
 
 You can use the xref:Provisioning_Virtual_Machines_in_VMware_vSphere-Adding_VMware_vSphere_Images_on_the_Satellite_Server[] section to add the image to {Project}.
 
+.Configuring {SmartProxy} to forward user data template
+
+{Project} must recognize the client IP address in order to serve correct template payload and when clients connects directly to {ProjectServer}, remote IP of the HTTP request can be used. However, when {Project} is deployed with {SmartProxy} templates feature, the callback URL is set to {SmartProxy} endpoint which forwards to the {ProjectServer} with HTTP header information X-Forwarded-For.
```

Comment:
Also, this description differs from what is in the web UI. In the web UI I have the following:
`If Satellite is running behind Passenger or a remote load balancer, the IP should be set here. This is a regular expression, so it can support several load balancers, i.e: (10.0.0.1|127.0.0.1)`
Is this expected?

Reply:
> Is this expected?

Yes, the feature was initially written for something else and now it's used for other features. We should probably reword that in core.

To your proposal, you kinda repeat "{Project} is deployed with the {SmartProxy} templates feature" twice which is a bit confusing. I suggest:

If you deploy {Project} with the {SmartProxy} templates feature, you must configure {Project} to recognize hosts' IP addresses forwarded via X-Forwarded-For HTTP header to serve correct template payload.

Reply:
```suggestion
If you deploy {Project} with the {SmartProxy} templates feature, you must configure {Project} to recognize hosts' IP addresses forwarded over the X-Forwarded-For HTTP header to serve correct template payload.
```
I am good with your suggestion